ontology/yaml/resources/HVAC/entity_types/TK.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+### Tanks ###
+
+TK_SWTC_PHWTC:
+  description: "Tank with water temperature monitoring"
+  is_canonical: true
+  implements:
+  - TK  
+  - SWTC
+  - PHWTC
+  
+TK_TLM:  
+  description: "Tank with water level monitoring"
+  is_canonical: true
+  implements:
+  - TK
+  - TLM 
+
+TK_SPR_PLPM:  
```

Comment:
Remove sprinkler reference, its unnecessary.

Reply:
Changed from TK_SPR_PLPM  to TK_PLPM

Reply:
deleted the canonical...